backend/api/models/DocumentClass.py

```diff
@@ -11,11 +11,16 @@ class DocumentClass(Mixin, db.Model):
     id = db.Column(db.String, unique=True, primary_key=True)
     name = db.Column(db.String, unique=True)
     description = db.Column(db.String, unique=False, nullable=True)
+    example = db.Column(db.String, unique=False, nullable=True)
+    # an example of this Document Class, represented by a shared Box link
 
-    def __init__(self, name, description=None):
+    def __init__(self, data):
```

Comment:
I changed the `__init__()` for all models from taking in parameters individually to taking in a dictionary containing the parameters as a result of the shift to FormData for all POST requests (covered later). If the method was left as is, then some parameters would be returned as `"name": ["Angad"]` instead of `"name": "Angad"`, or similar.